modules/access-control/src/access-control-context-provider.tsx

```diff
@@ -230,7 +230,23 @@ export const AccessControlContext: FunctionComponent<PropsWithChildren<AccessCon
                     featureConfig?.organization_roles,
                     featureConfig?.organization_roles?.scopes?.create,
                     allowedScopes
-                )
+                ),     
+                [ AccessControlConstants.EVENT_CONFIG_READ ] : hasRequiredScopes(
```

Comment:
Did we add the relevant config to the `deployment.config.json` file? And do we have the relevant j2 templating?

Reply:
J2 templating and config mapping can be added to the solution which is implementing this event publishing feature (Using the extension model). We provide support for extended feature declarations.

https://github.com/wso2/identity-apps/commit/aaeb789bfad20739fb52ba4105c3e84b30034a4b

Reply:
@thivi yeah here : 
https://github.com/wso2-enterprise/asgardeo-app-extensions/pull/2189/files#diff-adf8116735540d3bca46ace6bcf7c73841e1b5ca8e5e90cd59b881f77ee2a4c7R535